src/directives/clickaway.js

```diff
@@ -0,0 +1,101 @@
+/* eslint-disable no-constant-condition */
+// #taken from here https://github.com/simplesmiler/vue-clickaway
+import Vue from 'vue/dist/vue'
+
+let HANDLER = '_vue_clickaway_handler'
+
+function checkPassiveSupport () {
+  let supportsPassive = false
+  try {
+    let opts = Object.defineProperty({}, 'passive', {
+      get: function () {
+        supportsPassive = true
+      }
+    })
+    window.addEventListener('testPassive', null, opts)
+    window.removeEventListener('testPassive', null, opts)
+  } catch (e) {}
+
+  return supportsPassive
+}
+
+function bind (el, binding, vnode) {
+  unbind(el)
+
+  let vm = vnode.context
+
+  let callback = binding.value
+  if (typeof callback !== 'function') {
+    if ('development' !== 'production') {
+      Vue.util.warn(
+        'v-' + binding.name + '="' +
+        binding.expression + '" expects a function value, ' +
+        'got ' + callback
+      )
+    }
+    return
+  }
+
+  // @NOTE: Vue binds directives in microtasks, while UI events are dispatched
+  //        in macrotasks. This causes the listener to be set up before
+  //        the "origin" click event (the event that lead to the binding of
+  //        the directive) arrives at the document root. To work around that,
+  //        we ignore events until the end of the "initial" macrotask.
+  // @REFERENCE: https://jakearchibald.com/2015/tasks-microtasks-queues-and-schedules/
+  // @REFERENCE: https://github.com/simplesmiler/vue-clickaway/issues/8
+  let initialMacrotaskEnded = false
+  setTimeout(function () {
+    initialMacrotaskEnded = true
+  }, 0)
+
+  el[HANDLER] = function (ev) {
+    // @NOTE: this test used to be just `el.contains`, but working with path is better,
+    //        because it tests whether the element was there at the time of
+    //        the click, not whether it is there now, that the event has arrived
+    //        to the top.
+    // @NOTE: `.path` is non-standard, the standard way is `.composedPath()`
+    let panel = document.getElementById('alpheios-panel-inner') ? document.getElementById('alpheios-panel-inner') : null
+    let popup = document.getElementById('alpheios-popup-inner') ? document.getElementById('alpheios-panel-inner') : null
```

Comment:
This is a copy/paste bug .. second reference in document.getElementById in line 58 should be alpheios-popup-inner

```
let popup = document.getElementById('alpheios-popup-inner') ? document.getElementById('alpheios-popup-inner') : null
```